src/leappto/cli.py

```diff
@@ -494,8 +527,8 @@ def _rsync():
                                  .format(rsync_dir, ' '.join(self.target_cfg), self.target_addr)
                     Popen(shlex.split(target_cmd)).wait()
 
-                # temporary, after task with different names for containers should be removed
-                shutil.rmtree(rsync_dir)
+                    # temporary, after task with different names for containers should be removed
+                    shutil.rmtree(rsync_dir)
```

Comment:
I think this should be solved by picking unique name for the container (and hence the directory in which the files reside). But I'll wait for @vinzenz to clarify.